src/compositor/meta-surface-actor.c

```diff
@@ -364,6 +364,12 @@ meta_surface_actor_is_unredirected (MetaSurfaceActor *self)
   return META_SURFACE_ACTOR_GET_CLASS (self)->is_unredirected (self);
 }
 
+void
+meta_surface_actor_set_cull_back_face (MetaSurfaceActor *self, gboolean cull)
+{
```

Comment:
Again, this should have a `g_return_if_fail()` to check the `self` argument, and the `cull` argument should go on a separate line.